CHANGELOG.md

```diff
@@ -16,6 +17,7 @@ Changelog for NeoFS Node
 ### Updated
 
 ### Updating from v0.31.0
+`neofs-cli object put`'s object ID output has changed from "ID" to "OID"  
```

Comment:
Why did you wrote it here? I thought we reserve this for instructions on what to do.

Reply:
i thought that it should contain any information about using NeoFS components by sysadmins: configuration changes, the output of CLI commands (could be used in automated tests), etc

but yes, find out that we never wrote that here. dropped